docs/start.md

```diff
@@ -5,10 +5,8 @@ If you have any questions, please reach out to the TIPS team in our dedicated sl
 If you feel like these docs can be improved in any way, feel free to open a PR and assign someone from TIPS. No ticket necessary.
 
 # Index
-* [Setting Up](./setup.md)
+* [Using Genny (Start here!)](./using.md)
```

Comment:
Also consider incorporating or otherwise using the hitchhiker's guide to the codebase for genny internals (perhaps in `developing.md`) https://github.com/mongodb/genny/blob/master/docs/HitchhikersGuide.md